### PR TITLE
Feature/38 enhance element property getters logging

### LINKED ISSUE #38: Enhance logging for element property getters
Log values got from getText() and getAttribute(). Log failed waitings in DefaultElementStateProvider (as it was done in CachedElementStateProvider)

### PR DESCRIPTION
**Pull request by mialeska**
### PR Details

- Added logPageSource parameter to settings. Covered logPageSource functionalityWithTests
- Added Configuration property to ILocalizedLogger
- Added logging of values for getAttribute() and getText() of Element
- Enhance LocalizationManager to get value from core if the key is missed in passed assembly
- Added logging to ElementStateProviders' waiting functions

#### Related Issue Link: 

Closes #38 

#### Checklist

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: src/main/java/aquality/selenium/core/elements/DefaultElementStateProvider.java [hunk 1]
```diff
@@ -86,4 +110,21 @@ public boolean waitForEnabled(Duration timeout) {
...
+
+    private boolean doAndLogWaitForState(BooleanSupplier waitingAction, String conditionKeyPart, Duration timeout)
+    {
+        if (Duration.ZERO.equals(timeout))
+        {
+            return waitingAction.getAsBoolean();
+        }
+
+        logElementState("loc.wait.for.state", conditionKeyPart);
+        boolean result = waitingAction.getAsBoolean();
```
Comment (DmitryBogatko) [inline src/main/java/aquality/selenium/core/elements/DefaultElementStateProvider.java:122]: can we move this var at the first line in the method and use it for 'ZERO' branch too

File: src/main/java/aquality/selenium/core/elements/DefaultElementStateProvider.java [hunk 2]
```diff
@@ -86,4 +110,21 @@ public boolean waitForEnabled(Duration timeout) {
     public boolean waitForNotEnabled(Duration timeout) {
         return isElementInDesiredCondition(elementNotEnabled(), timeout);
     }
+
+    private boolean doAndLogWaitForState(BooleanSupplier waitingAction, String conditionKeyPart, Duration timeout)
```
Comment (DmitryBogatko) [inline src/main/java/aquality/selenium/core/elements/DefaultElementStateProvider.java:114]: looks strange that logging behaviour depends on timeout value

File: src/test/java/tests/applications/browser/AqualityServices.java [hunk 3]
```diff
@@ -35,6 +40,13 @@ public static Injector getServiceProvider() {
         return getInstance().getInjector();
     }
 
+    public static <T extends AqualityModule<ChromeApplication>> void initInjector(T module) {
```
Comment (DmitryBogatko) [inline src/test/java/tests/applications/browser/AqualityServices.java:43]: is it fix of some bug? or how is it related with logger?

File: src/main/java/aquality/selenium/core/elements/DefaultElementStateProvider.java (hunk 2 above)
Comment (mialeska) [reply src/main/java/aquality/selenium/core/elements/DefaultElementStateProvider.java:114]: it does, we don't want to log "wait" if we don't wait (ZERO timeout).

File: src/test/java/tests/applications/browser/AqualityServices.java (hunk 3 above)
Comment (mialeska) [reply src/test/java/tests/applications/browser/AqualityServices.java:43]: it is related with tests cleanup. it's a test class.

File: src/main/java/aquality/selenium/core/elements/DefaultElementStateProvider.java (hunk 1 above)
Comment (mialeska) [reply src/main/java/aquality/selenium/core/elements/DefaultElementStateProvider.java:122]: no we can not, the order of logging-action-logging does matter here

**Pull request merged**